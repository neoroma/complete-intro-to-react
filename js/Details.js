import React from 'react'

export class Details extends React.Component {
    render() {
        return (
            <h1>OmgLolHi this is a details page hahaha</h1>
        )
    }
}
