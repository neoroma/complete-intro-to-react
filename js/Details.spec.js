/* global test, expect */

import React from 'react'
import {shallow} from 'enzyme'
import {shallowToJson} from 'enzyme-to-json'
import {Details} from './Details'

test('Details component snapshot test', () => {
    const component = shallow(<Details />)
    const tree = shallowToJson(component)
    expect(tree).toMatchSnapshot()
})
