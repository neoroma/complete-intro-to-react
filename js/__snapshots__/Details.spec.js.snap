// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details component snapshot test 1`] = `
<h1>
  OmgLolHi this is a details page hahaha
</h1>
`;
